crowdaction/models.go

```diff
@@ -10,5 +10,5 @@ type Crowdaction struct {
 	DateStart     string `json:"date_start,omitempty"`
 	DateEnd       string `json:"date_end,omitempty"`
 	DateLimitJoin string `json:"date_limit_join,omitempty"`
-	Code          string `json:"code,omitempty"`
+	PasswordJoin  string `json:"code,omitempty"`
```

Comment:
The GoLang code uses "password" the JSON uses "code".  
We should probably be consistent and refer to it by the same name everywhere.